distribution/helm/vector-agent/values.yaml

```diff
@@ -141,11 +151,11 @@ vectorSink:
   # The name to use for the "built-in" vector sink.
   sinkId: vector_sink
   # Inputs of the built-in vector sink.
-  inputs: null  # ["my_input_1", "my_input_2"]
+  inputs: null # ["my_input_1", "my_input_2"]
   # The host of the Vector to send the data to.
-  host: null  # "vector.internal"
+  host: null # "vector.internal"
   # The port of the Vector to send the data to.
-  port: null  # 9000
+  port: null # 9000
   # Additional config to embed at the vector sink.
```

Comment:
These are conflicting with `ct` linter, see https://github.com/timberio/vector/pull/6021
Let's not change those at this PR.